common/kotlinx-coroutines-core-common/src/Dispatched.kt

```diff
@@ -227,6 +234,12 @@ internal interface DispatchedTask<in T> : Runnable {
     }
 }
 
+internal fun DispatchedContinuation<Unit>.yield(): Boolean {
```

Comment:
1. Maybe rename to `yieldUndispatched` & document the meaning of boolean result. In the context of this particular file this extension function is hard to understand.
2. Also refactor to idiomatic expression body, please, since this function does not do anything else but delegates to `UndispatchedEventLoop.execute` and this intent should be immediately clear.